src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -39,25 +39,24 @@ Factors affecting access to the migration wizard:
 
 ## Before you start [#before-beginning]
 
-The more complex your organization is, the more it may help you to first understand our [new user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts). A brief summary: 
-* Users reside within a container known as an “authentication domain,” which governs how those users are added to New Relic (manually with the UI vs. automatically with a SCIM-based identity provider), and how they log in (manually with email/password vs. using SAML SSO), and other settings. Typically organizations have just one or two domains: one with the default manual settings and one for the more automatic methods.  
-* Users can be assigned to one or more groups (for example, our default **Admin** group or a custom group like **Contractors**). For large organizations, users are often in multiple groups. 
-* When you want to add new groups, or give groups access to more accounts, you must create an access grant. This assigns a group access to: 1) a specific role and 2) a specific account. For example, you may give a Contractors group access to our default **All product admin** role on one or more of your accounts, or give that group a custom role (a role being a custom set of [capabilities](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts)). 
+At a certain point in the user migration procedure, you'll have a choice to either grant all your users access to all accounts in your organization, or to customize which accounts your users have access to (aka, assign **access grants**). 
 
-To read more about these concepts, see [User management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#understand-concepts).
+The more you need to partition your users' access to accounts or different roles, the more it will help you to understand some basics about our newer user management concepts. Here's a brief summary: 
 
+* Users reside within a container known as an “authentication domain,” which governs how those users are added to New Relic (manually with the UI vs. automatically with a SCIM-based identity provider), and how they log in (manually with email/password vs. using SAML SSO), and other settings. Typically organizations have just one or two authentication domains: one with the default manual settings and one for the more automatic methods.  
```

Comment:
There's a lot going on in this first bullet (two parentheticals in one sentence feels like a lot to me... Maybe we can break it up a little...

something like:

"Users are grouped in a container called an "authentication domain". The domain governs how users are added to New Relic (manually with the UI or automatically with a SCIM-based identity provider). It also governs how users log in (manually with email/password or using SAML SSO)..."

I dropped "other settings" there, because I'm not sure how useful that is, unless specific other settings are mentioned.